jscomp/core/lam_stats_export.mli

```diff
@@ -35,5 +35,6 @@ val export_to_cmj :
   Js_cmj_format.effect ->
   Lam.t Map_ident.t ->
   Ext_js_file_kind.case ->
+  (* FIXME: this is a leaky abstraction *)
```

Comment:
why?

Reply:
lam_stats_export is not only exporting data to optimizations but the full cmj, which now includes the full JS